pkg/cmd/label/clone.go

```diff
@@ -0,0 +1,141 @@
+package label
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/internal/ghrepo"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/cli/cli/v2/utils"
+	"github.com/spf13/cobra"
+)
+
+type cloneOptions struct {
+	BaseRepo   func() (ghrepo.Interface, error)
+	HttpClient func() (*http.Client, error)
+	IO         *iostreams.IOStreams
+
+	SourceRepo ghrepo.Interface
+	Force      bool
+}
+
+func newCmdClone(f *cmdutil.Factory, runF func(*cloneOptions) error) *cobra.Command {
+	opts := cloneOptions{
+		HttpClient: f.HttpClient,
+		IO:         f.IOStreams,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "clone <source [HOST/]OWNER/REPO>",
```

Comment:
For consistency with other commands lets use `repository` rather than the accepted format. 
```suggestion
		Use:   "clone <source-repository>",
```

Reply:
Something like that was my original thought, but my worry was that there's no way to specify documentation for arguments; though, I suppose I could just put something in the description. I wanted to make it clear that they could use the same "[host/]owner/repo" that they can for `-R`.

Or do you think samples alone would be enough?

Long-term since you're already overriding doc output for cobra anyway, should arguments get their own documentation? `gh` uses quite a few arguments and some could benefit users- especially new users - with more description.

Reply:
I think that most our users are familiar with the format we accept for repo arguments, and if they do enter an incorrect format the error message displays the formats we accept. Additionally an example or two should be sufficient enough that we don't need to add anything to the description explicitly.

I think arguments getting their own documentation sounds like good idea and perhaps something we can address in the future.